src/itemCheckbox/ItemCheckbox.tsx

```diff
@@ -16,6 +16,7 @@ export interface ItemCheckboxProps extends A11yProps {
   readonly name: string
   readonly data?: string
   readonly className?: string
+  readonly leftAddon?: React.ReactNode
```

Comment:
Should i rename this `icon` to be more explicit? For now i just went with the same name as the base item, but as it's internal, we might as well not stay strict to its props. WDYT?

Reply:
A name I often see for this kind of thing is "adornment"

Reply:
I would keep a name that match the actual value that can be passed. So leftAddon will be good here IMO (unless you find a way to type the param value to be only an icon).